Pipfile

```diff
@@ -25,6 +25,7 @@ redis = "~=3.5"
 requests = ">=2.20"
 rq = "~=1.8"
 # run `pipenv install --dev` to get the packages below in your env
+pytest = "*"
```

Comment:
pytest is already included in the dev packages, to install them you need just to run `pipenv install --dev`